fury/ui/containers.py

```diff
@@ -737,8 +740,10 @@ def _get_size(self):
     def update_tabs(self):
         """Update position, size and callbacks for tab panels."""
         self.tab_panel_size = (self.size[0] // self.nb_tabs, int(0.1 * self.size[1]))
-
-        tab_panel_pos = [0.0, 0.9]
+        if self.tab_bar_pos.lower() == "top":
+            tab_panel_pos = [0.0, 0.9]
+        elif self.tab_bar_pos.lower() == "bottom":
+            tab_panel_pos = [0.0, 0.0]
```

Comment:
What happen if it is not "top" or "bottom" for `self.tab_bar_pos`

Reply:
Ohh Sorry my bad... I missed that. In case if it is not 'top' or 'bottom' I should throw a value error

Reply:
you should use warnings module

`from warning import warn.` then `warn(my_msg)`